Code/Framework/AzCore/AzCore/variadic.h

```diff
@@ -127,34 +127,150 @@ static_assert(AZ_VA_NUM_ARGS(1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15,
+#define AZ_JOIN_EACH_98( param, ...) AZ_JOIN_TWO_INTERNAL(param, AZ_JOIN_EACH_97(__VA_ARGS__))
+#define AZ_JOIN_EACH_99( param, ...) AZ_JOIN_TWO_INTERNAL(param, AZ_JOIN_EACH_98(__VA_ARGS__))
+#define AZ_JOIN_EACH_100(param, ...) AZ_JOIN_TWO_INTERNAL(param, AZ_JOIN_EACH_99(__VA_ARGS__))
+#define AZ_JOIN_EACH_101(param, ...) AZ_JOIN_TWO_INTERNAL(param, AZ_JOIN_EACH_100(__VA_ARGS__))
+#define AZ_JOIN_EACH_102(param, ...) AZ_JOIN_TWO_INTERNAL(param, AZ_JOIN_EACH_101(__VA_ARGS__))
+#define AZ_JOIN_EACH_103(param, ...) AZ_JOIN_TWO_INTERNAL(param, AZ_JOIN_EACH_102(__VA_ARGS__))
+#define AZ_JOIN_EACH_104(param, ...) AZ_JOIN_TWO_INTERNAL(param, AZ_JOIN_EACH_103(__VA_ARGS__))
+#define AZ_JOIN_EACH_105(param, ...) AZ_JOIN_TWO_INTERNAL(param, AZ_JOIN_EACH_104(__VA_ARGS__))
+#define AZ_JOIN_EACH_106(param, ...) AZ_JOIN_TWO_INTERNAL(param, AZ_JOIN_EACH_105(__VA_ARGS__))
+#define AZ_JOIN_EACH_107(param, ...) AZ_JOIN_TWO_INTERNAL(param, AZ_JOIN_EACH_106(__VA_ARGS__))
+#define AZ_JOIN_EACH_108(param, ...) AZ_JOIN_TWO_INTERNAL(param, AZ_JOIN_EACH_107(__VA_ARGS__))
+#define AZ_JOIN_EACH_109(param, ...) AZ_JOIN_TWO_INTERNAL(param, AZ_JOIN_EACH_108(__VA_ARGS__))
+#define AZ_JOIN_EACH_110(param, ...) AZ_JOIN_TWO_INTERNAL(param, AZ_JOIN_EACH_109(__VA_ARGS__))
+#define AZ_JOIN_EACH_111(param, ...) AZ_JOIN_TWO_INTERNAL(param, AZ_JOIN_EACH_110(__VA_ARGS__))
+#define AZ_JOIN_EACH_112(param, ...) AZ_JOIN_TWO_INTERNAL(param, AZ_JOIN_EACH_111(__VA_ARGS__))
+#define AZ_JOIN_EACH_113(param, ...) AZ_JOIN_TWO_INTERNAL(param, AZ_JOIN_EACH_112(__VA_ARGS__))
+#define AZ_JOIN_EACH_114(param, ...) AZ_JOIN_TWO_INTERNAL(param, AZ_JOIN_EACH_113(__VA_ARGS__))
+#define AZ_JOIN_EACH_115(param, ...) AZ_JOIN_TWO_INTERNAL(param, AZ_JOIN_EACH_114(__VA_ARGS__))
+#define AZ_JOIN_EACH_116(param, ...) AZ_JOIN_TWO_INTERNAL(param, AZ_JOIN_EACH_115(__VA_ARGS__))
+#define AZ_JOIN_EACH_117(param, ...) AZ_JOIN_TWO_INTERNAL(param, AZ_JOIN_EACH_116(__VA_ARGS__))
+#define AZ_JOIN_EACH_118(param, ...) AZ_JOIN_TWO_INTERNAL(param, AZ_JOIN_EACH_117(__VA_ARGS__))
+#define AZ_JOIN_EACH_119(param, ...) AZ_JOIN_TWO_INTERNAL(param, AZ_JOIN_EACH_118(__VA_ARGS__))
+#define AZ_JOIN_EACH_120(param, ...) AZ_JOIN_TWO_INTERNAL(param, AZ_JOIN_EACH_119(__VA_ARGS__))
+#define AZ_JOIN_EACH_121(param, ...) AZ_JOIN_TWO_INTERNAL(param, AZ_JOIN_EACH_120(__VA_ARGS__))
+#define AZ_JOIN_EACH_122(param, ...) AZ_JOIN_TWO_INTERNAL(param, AZ_JOIN_EACH_121(__VA_ARGS__))
+#define AZ_JOIN_EACH_123(param, ...) AZ_JOIN_TWO_INTERNAL(param, AZ_JOIN_EACH_122(__VA_ARGS__))
+#define AZ_JOIN_EACH_124(param, ...) AZ_JOIN_TWO_INTERNAL(param, AZ_JOIN_EACH_123(__VA_ARGS__))
+#define AZ_JOIN_EACH_125(param, ...) AZ_JOIN_TWO_INTERNAL(param, AZ_JOIN_EACH_124(__VA_ARGS__))
+
 
 // Joins 2 or more macro parameters together
-#define AZ_JOIN_VA_ARGS_IMPL(X, ...) X ## __VA_ARGS__
-#define AZ_JOIN_VA_ARGS(X, ...) AZ_JOIN_VA_ARGS_IMPL(X, __VA_ARGS__)
-
-
-// Out of all supported compilers, mwerks is the only one
```

Comment:
I had to look up mwerks

Reply:
I hoped this change m"werks" out for you